distribution/docs/src/main/resources/_contents/_sources/solrcloud-contents.adoc

```diff
@@ -0,0 +1,50 @@
+==== Solr Cloud
```

Comment:
I can't seem to find where this `adoc` file is being used or integrated anywhere.